src/ghost.js

```diff
@@ -68,7 +68,16 @@ class GhostThread extends EventEmitter {
    * @return    {Array<Object>}      users online
    */
   async getPosts () {
-    return [...this._backlog].map(msg => JSON.parse(msg)).sort((p1, p2) => p1.timestamp - p2.timestamp)
+    const _12hrInSecs = 12 * 60 * 60
+    const posts = [...this._backlog]
+      .map(msg => JSON.parse(msg))
+      .sort((p1, p2) => p1.timestamp - p2.timestamp)
+      .filter(msg => msg.timestamp > (Math.floor(Date.now() / 1000) - _12hrInSecs))
```

Comment:
Should we add this filter to before we send the backlog to other peers as well?

Reply:
When we send the backlog to other peers, we go use the `getPosts()` method as well, so we are sending them a filtered version.

```js
case 'request_backlog':
this.getPosts()
 .then(posts => this._sendDirect({ type: 'backlog_response', message: posts }, from))
 break
```

Reply:
Oh, sorry totally missed that!